app/scripts/templates/settings/change_password.mustache

```diff
@@ -18,7 +18,7 @@
         <label class="label-helper"></label>
         <input type="password" class="password" id="old_password" placeholder="{{#t}}Old password{{/t}}" required pattern=".{8,}" autofocus />
 
-        <input id="show-old-password" type="checkbox" class="show-password" aria-controls="old_password" tabindex="-1" data-novalue />
+        <input id="show-old-password" type="checkbox" class="show-password" aria-controls="old_password" tabindex="-1" data-novalue data-unsynchronized-input="true"/>
```

Comment:
`data-unsynchronized-input="true"` good suggestion here. It is a difficult to name this one. 
How about `data-unsynchronized-show="true"`? 

Using `-input` might confuse future developers, expecting actual keyboard input to be synchronized.